<launch>


  <arg name="cameraIntrinsicInput"  default="$(find camera_transform_publisher)/camera_intrinsic_color_multi_left.xml" />
  <arg name="cameraTopic"  default="/multicam_driver/left/rgb" />
  <arg name="cameraLinkName"  default="/camera_link_l" />
  <arg name="arMarkerName"  default="/ar_marker_l" />

  <node pkg="camera_transform_publisher" type="camera_transform_publisher_l" name="camera_transform_publisher_l" output="screen">
  <!-- handeye_calib_camodocal arg pass -->
      <param name="cameraIntrinsicInput" type="str"  value="$(arg cameraIntrinsicInput)" />
      <param name="cameraTopic" type="str"  value="$(arg cameraTopic)" />
      <param name="cameraLinkName" type="str"  value="$(arg cameraLinkName)" />
      <param name="arMarkerName" type="str"  value="$(arg arMarkerName)" />
      <param name="chessboardWidth"       type="int" value="7"/>
      <param name="chessboardHeight"      type="int" value="6"/>
      <param name="squareSize"    type="double"  value="0.15" />
  </node>

</launch>
